### ISSUE TITLE
Agenda view?

### ISSUE THREAD

**Issue by evandavis** [IvanMalison/org-projectile#51]
I'm using `org-projectile` in single-file mode (projects todos go under project-name headlines in `~/Org/projects.org`.) 

*Is there an easy way to view my todos filtered to the current project?* 

I'm happy to add this in userland, but is this something that could be added to the package?

**Comment by IvanMalison** [comment]
@evandavis use org categories? I believe that categories should be added to project headings automatically, but its been a while since I actually checked.

**Comment by evandavis** [comment]
The category is added automatically, but I'm stuck trying to get the name of the current project into the category filter from the agenda dispatcher. Writing it out, I think this might be a better question for StackExchange...

**Comment by IvanMalison** [comment]
> The category is added automatically, but I'm stuck trying to get the name of the current project into the category filter from the agenda dispatcher. Writing it out, I think this might be a better question for StackExchange...

Wdym by the current project. In the agenda dispatcher it may be hard to get some notion of the current project.

**Comment by evandavis** [comment]
What I mean is this:

1. I'm editing a file in a known project, `my-project`. 
2. From that buffer, I do `C-c a` to bring up the agenda dispatcher.
3. I press `p` to bring up `my-project` todos.

-----

I think what I actually want is for `org-projectile` to export an agenda view that I can bind to my projectile-map, like
```lisp
:bind ("s-p A" . org-projectile-project-agenda) // agenda view for this project's todos

**Comment by IvanMalison** [comment]
Right, the problem is that you want an agenda command that is contextual i.e. it does something different depending on your current editing state. That's just not really possible as far as I remember with org-agenda.

That said, it's very easy to filter by category. It should also be possible to write a small custom bit of elisp to do what you want.

**Comment by colonelpanic8** [comment]
@evandavis did you ever get what you wanted here?

**Comment by colonelpanic8** [comment]
Implemented on `master` in `6a95fb9`. There is now an `org-project-capture-agenda-for-current-project` command that opens an agenda view scoped to the current project across the supported strategy types. Closing.
